newIDE/app/src/EventsSheet/InstructionEditor/InstructionOrExpressionSelector/index.js

```diff
@@ -164,6 +170,18 @@ export default class InstructionOrExpressionSelector<
                         initiallyOpenedPath: this.initialInstructionTypePath,
                         selectedItemRef: this._selectedItem,
                       })}
+                      {onClickMore && (
+                        <Line justifyContent="center">
+                          <RaisedButton
+                            primary
+                            icon={<Add />}
+                            onClick={onClickMore}
+                            label={<Trans>Add new behavior</Trans>}
```

Comment:
Would "Add a new behavior to the object" work? Other "Add a new behavior".

Reply:
It reads better. I will check the display is not cumbersome on mobile

Reply:
I went for "Add a new behavior to the object" here, and "Add a new extension to the project" for free instructions

Reply:
Oups, didn't see your suggestion below, I will use "Search for new actions in extensions"